syncd/Syncd.cpp

```diff
@@ -4181,15 +4181,36 @@ sai_status_t Syncd::setUninitDataPlaneOnRemovalOnAllSwitches()
 
         auto strRid = sai_serialize_object_id(rid);
 
-        auto status = m_vendorSai->set(SAI_OBJECT_TYPE_SWITCH, rid, &attr);
+        sai_attr_capability_t attr_capability = {};
 
-        if (status != SAI_STATUS_SUCCESS)
+        sai_status_t queryStatus;
+
+        queryStatus = sai_query_attribute_capability(rid,
```

Comment:
don't use SAI api directly, use m_vendorSai

Reply:
Apologies for the delay. I have addressed this comment. Please take a look.